src/Apps/FeatureAKG/Components/FeaturedArticles.tsx

```diff
@@ -31,14 +31,11 @@ interface FeaturedArticlesProps {
 const FeaturedArticles: React.FC<FeaturedArticlesProps> = props => {
   const articles = [...props.articles]
 
-  if (!articles?.length) {
-    return null
-  }
-
   const firstArticle = articles.shift()
   const firstArticleImage = firstArticle?.thumbnailImage?.cropped
```

Comment:
This is really the only line here that would be suspect but it's already guarded.